src/components/Events/MobileDay/index.jsx

```diff
@@ -0,0 +1,22 @@
+export default function MobileDay({ index,text, events }) {
+  return (
+    <div className="group outline-none" tabIndex={index}>
+      <div className="group bg-primary flex justify-between px-6 py-4 items-center text-gray-500 transition ease duration-500 cursor-pointer">
+        <h3 className="text-white group-focus:text-black transition ease duration-500">
```

Comment:
I think it would look better if the text stayed white, but it would look nice if you darkened the background to primary-dark on the tab that was selected.

Reply:
It does look better this way! Fixed!